context_chat_backend/chain/ingest/doc_loader.py

```diff
@@ -22,14 +22,10 @@
 
 def _temp_file_wrapper(file: BinaryIO, loader: Callable, sep: str = '\n') -> str:
 	raw_bytes = file.read()
-	with tempfile.NamedTemporaryFile(mode='wb', delete=False) as tmp:
+	with tempfile.NamedTemporaryFile(mode='wb') as tmp:
 		tmp.write(raw_bytes)
 		docs = loader(tmp.name)
 
-		if not tmp.delete:
-			import os
-			os.remove(tmp.name)
```

Comment:
What's this about? :)

Reply:
I don't remember why it was like that, delete is true by default so temp files should be cleared on context exit

Reply:
I vaguely remember we once wanted to not delete the file immediately because we still needed it, but it seems like that's no longer the case :shrug: